test/unit_test/character_sheet_unit_tests/RaceUnitTests.java

```diff
@@ -0,0 +1,27 @@
+package unit_test.character_sheet_unit_tests;
+
+import static org.junit.Assert.*;
+
+import org.junit.Test;
+
+import rpg_database.character_sheet.Background;
+import rpg_database.character_sheet.CharacterSheet;
+import rpg_database.character_sheet.Fields;
+import rpg_database.character_sheet.Race;
+
+public class RaceUnitTests {

```

Comment:
Nem látok teszteket a bázissebességekre. Azt nem ebben a taskban implementálod? Sztem hozzátartozik a Race osztályhoz mint olyanhoz, még ha ebben a taskban nem is használod semmire.

Reply:
ok